src/components/DeleteComment.jsx

```diff
@@ -0,0 +1,24 @@
+import React, {useState} from 'react'
+import DeleteOutlineOutlinedIcon from '@mui/icons-material/DeleteOutlineOutlined';
+import * as api from '../utils/api'
+import './comments.css'
+
+const DeleteComment = ({comment_id, renderKey, setRenderKey}) => {
+
+  const [isDeleted, setIsDeleted] = useState(false)
+    const handleClick = (comment_id) => {
+      setIsDeleted(true)
+      api.deleteComments(comment_id).then(() => {
+        setRenderKey(!renderKey);
+        setIsDeleted(false)
```

Comment:
You need to update the comments locally, instead of doing an api request again to get the comments. 
If someone else is using the site, and also delete something at the same time, it would be confusing UI as it might look like you've deleted multiple ones. 
You need to delete, then update it yourself. The next time they refresh the page, they can then see what other people have deleted.